### DISCUSSION TITLE
MQTT Protocol Version 4/5

### DISCUSSION THREAD

**Discussion by WojtekWro** [Koenkk/zigbee2mqtt#16880] [General]
Hi Guys, 
Today I use MQTT protocol version 4 in my network. 
I am wondering if switch to version 5 is good idea and how it impacts my network. 
What is your opinion?

**Comment by ovizii** [comment]
Did you ever figure out more about this topic?

**Comment by Ra72xx** [comment]
I switched to version 5 with Mosquitto as broker a few months and do not really recognize a difference. I would also be interested what the difference should be (or why version 5 isn't the default, it's not really a brand new spec).
